src/public/index.html

```diff
@@ -14,8 +14,12 @@
 <script src="/socket.io/socket.io.js"></script>
 <script>
   const socket = io('http://localhost:3000');
-  socket.on('approve_reject', (name) => {
+  const rr = ['approve_reject_client','booking_client','trip_request_client'];
+  rr.forEach((ev)=>{
+    socket.on(ev, (name) => {
```

Comment:
It is good but It should be better when you rename this variable `rr` to a meaningful name.